docs/tensorboard.md

```diff
@@ -0,0 +1,29 @@
+# How to view metrics on tensorboard
```

Comment:
I assume that same should work on cluster using tp run.
Could you call it out here as well